src/libs/ajax/DatasetBuilder.ts

```diff
@@ -1,11 +1,8 @@
 // Types that can be used to create a criteria.
 import _ from 'lodash/fp';
-import { Ajax } from 'src/libs/ajax';
```

Comment:
Do you think this file should still live in the `ajax` folder? Or should we move it into a libs/utils folder? I think we might want to not have it stored in `src/libs/ajax` anymore if it isn't making ajax calls.

Reply:
yeah good call